### DISCUSSION TITLE
After concatenating dates, resultant data array looks wonky

### DISCUSSION THREAD

**Discussion by advythr** [earthlab-education/earth-analytics-02-python-2024#14] [Q&A02 Land Cover Classification]
Hi all,

I have a list ```all_rasters_list``` containing 3 data arrays for each date (I am testing this code with a subset of the data first). Each date's data array is of the structure:

```
Coordinates:
date
band
x
y
spatial_ref
Indexes:
date
band
x
y
```

Where band is a coordinate.

When I select a date and plot a band, the data looks fine:
<img width="1488" alt="Screen Shot 2024-03-09 at 10 01 16 PM" src="https://github.com/earthlab-education/earth-analytics-02-python-2024/assets/83184164/24876186-a2dd-434a-9a6d-7d8cf123a434">

However, if I plot an image from another date, the images seem to have very different x scales?

<img width="1478" alt="Screen Shot 2024-03-09 at 10 02 08 PM" src="https://github.com/earthlab-education/earth-analytics-02-python-2024/assets/83184164/38130a1e-2c05-4afc-a286-e293f803aae4">

And then, when I concatenate along the date dimension: ```all_rasters_da = xr.concat(all_rasters_list, dim='date')```

I get a really weird output if I select the same date and plot the same band:
<img width="1486" alt="Screen Shot 2024-03-09 at 10 01 33 PM" src="https://github.com/earthlab-education/earth-analytics-02-python-2024/assets/83184164/a931d82e-07e0-44bf-a135-92a63b6bdb94">

In other words, when I plot the same band and date image before concatenating dates, and after concatenating dates, the data seems to be corrupted?

Anyone have an idea of what might be the issue here? Thanks!

System info:
Mac M1 Max 2021. Jupyter Lab within VS Code. Earth Analytics Python environment, Python 3.10.13.

@eculler
